### DISCUSSION TITLE
The FINOS community can engage in multi channel chatrooms

### DISCUSSION THREAD

**Discussion by maoo** [finos/open-developer-platform#195] [General]
- [ ] Validate that banks can use Symphony chat and can communicate with other firms
- [ ] Validate that individual contributors can use my.symphony.com
- [ ] Validate that banks allow their Symphony pod to communicate with my.symphony.com
- [ ] Spike on multi-lateral chat bridge between Symphony and Slack
  - https://matrix.org/
    - https://riot.im - clients
    - https://matrix.org/ - standards
    - https://matrix.org/bridges - bridges
    - https://www.modular.im/ - matrix hosting (paid)

**Comment by mcleo-d** [comment]
This discussion has moved back to https://github.com/finos/open-developer-platform/issues/48 following a question made by the FINOS community which is now being investigated by the FINOS team ... https://github.com/finos/community/issues/100
